to_be_transformed

```diff
@@ -111,9 +111,9 @@ network.floating_ip.deallocate specific-payload
 network.floating_ip.disassociate specific-payload
 scheduler.select_destinations.end specific-payload
 scheduler.select_destinations.start specific-payload
-servergroup.addmember specific-payload
-servergroup.create specific-payload
-servergroup.delete specific-payload
+server_group.addmember specific-payload
```

Comment:
it should be server_group.add_member instead to follow the convention we set up in aggregates